src/main/java/util/csvparsing/CsvMonumentConverter.java

```diff
@@ -137,6 +142,91 @@ else if (dateArray.length == 3) {
         return result;
     }
 
+    /**
+     * Static method for converting a specified Monument into a CSV row
+     * @param monument - Monument to convert into a CSV row
+     * @return String - CSV row representing the specified Monument
+     */
+    public static String convertMonument(Monument monument) {
+        String csvRow = "";
+
+        // Column 1: ID
+        csvRow += convertToCsvCell(monument.getId());
+
+        csvRow += ",";
+
+        // Column 2: Title
+        csvRow += convertToCsvCell(monument.getTitle());
+
+        csvRow += ",";
+
+        // Column 3: Artist
+        csvRow += convertToCsvCell(monument.getArtist());
+
+        csvRow += ",";
+
+        // Column 4: Date
+        // Format: "dd-mm-yyyy"
+        csvRow += convertToCsvCell(monument.getDate());
+
+        csvRow += ",";
+
+        // Column 5: Material
+        csvRow += convertToCsvCell(monument.getMaterial());
+
+        csvRow += ",";
+
+        // Column 6: Latitude
+        csvRow += convertToCsvCell(monument.getLat());
+
+        csvRow += ",";
+
+        // Column 7: Longitude
+        csvRow += convertToCsvCell(monument.getLon());
+
+        csvRow += ",";
+
+        // Column 8: City
+        csvRow += convertToCsvCell(monument.getCity());
+
+        csvRow += ",";
+
+        // Column 9: State
+        csvRow += convertToCsvCell(monument.getState());
+
+        csvRow += ",";
+
+        // Column 10: Address
+        csvRow += convertToCsvCell(monument.getAddress());
+
+        csvRow += ",";
+
+        // Column 11: Inscription
+        csvRow += convertToCsvCell(monument.getInscription());
+
+        csvRow += ",";
+
+        // Column 12: Description
+        csvRow += convertToCsvCell(monument.getDescription());
+
+        csvRow += ",";
+
+        // Column 13: Tags
+        csvRow += convertTagsToCsvCell(monument.getTags());
+
+        csvRow += ",";
+
+        // Column 14: Contributors
+        csvRow += convertContributionsToCsvCell(monument.getContributions());
+
+        csvRow += ",";
```

Comment:
I would recommend `String.join(",", listOfCells)` although I realize this code is pretty temporary

Reply:
No :)